@rspack/dev-server fails to declare it's dependency on @types/ws

I'm trying to upgrade to rspack 2.0 and @rspack/dev-server causes typecheck failures due to the undeclared dependency on ws - it's declared as a devDependency but is used in the published types.

node_modules/@rspack/dev-server/dist/index.d.ts(32,42): error TS2307: Cannot find module 'ws' or its corresponding type declarations.
node_modules/@rspack/dev-server/dist/index.d.ts(33,54): error TS2307: Cannot find module 'ws' or its corresponding type declarations.

@rspack/cli just solved a similar problem via bundling their dts (web-infra-dev/rspack#13807), not sure if that solution applies here or if ws actually needs to be referenced in the exported types, in which case @types/ws would be needed as a dependency and not a devDependency

--- a/rslib.config.ts
+++ b/rslib.config.ts
@@ -7,7 +7,13 @@ export default defineConfig({
       syntax: 'es2023',
       dts: {
         bundle: {
-          bundledPackages: ['chokidar', 'readdirp', 'connect-next', 'ws'],
+          bundledPackages: [
+            'chokidar',
+            'readdirp',
+            'connect-next',
+            'ws',
+            '@types/ws',
+          ],
         },
       },
       source: {

